webpack/components/extensions/HostDetails/DetailsTabCards/SystemPropertiesCardExtensions.js

```diff
@@ -35,6 +36,71 @@ SystemPropertiesCardSubscription.defaultProps = {
   hostDetails: {},
 };
 
+export const SystemPropertiesCardVirtualization = ({ hostDetails }) => {
+  if (!hostDetails?.subscription_facet_attributes) return null;
+
+  const {
+    virtual_guests: virtualGuests,
+    hypervisor,
+    registered_through: registeredThrough,
+    virtual_host: virtualHost,
+  } = hostDetails.subscription_facet_attributes;
+  const virtualGuestIds = virtualGuests.map(guest => `name = ${guest.name}`).join(' or ');
+
+  return (
+    <>
+      {hypervisor &&
+      <DescriptionListGroup>
+        <DescriptionListTerm>{__('Virtual guests')}</DescriptionListTerm>
+        <DescriptionListDescription>
+          <a href={`/hosts?search=${virtualGuestIds}`}>
```

Comment:
Whenever there can be spaces, it seems correct to encode the URI.

```suggestion
          <a href={`/hosts?search=${encodeURI(virtualGuestIds)}`}>
```